### PR TITLE
feat(docker): add info on docker image version

### PR DESCRIPTION
**Pull request by baptistemesta**
Closes [RUNTIME-289](https://bonitasoft.atlassian.net/browse/RUNTIME-289)

### CODE REVIEW

File: modules/ROOT/pages/bonita-docker-installation.adoc [hunk 1]
```diff
@@ -7,30 +7,45 @@ This guide assumes you have a working docker environment.
...
+* `latest`: Points to the latest version of the platform
+* `pass:a[{bonitaVersion}]`: Points to the latest maintenance version of the pass:a[{bonitaVersion}] version
+* `pass:a[{bonitaVersion}]-XXXX`: Tag of a maintenance version of pass:a[{bonitaVersion}]
+
+
+[WARNING]
+====
+`latest` and `pass:a[{bonitaVersion}]` are tags that change each time a new version of the platform is released.
+It is safer to use the more complete version tag `pass:a[{bonitaVersion}]-XXXX` when scripting deployments to ensure
+the version does not change.
```
Comment (educhastenier) [inline modules/ROOT/pages/bonita-docker-installation.adoc]:
```suggestion
the binaries do not change.
```

File: antora.yml [hunk 2]
```diff
@@ -6,7 +6,6 @@ asciidoc:
     bonitaVersion: 2021.1
     bonitaTechnicalVersion: '7.12.1' # latest public community version
     javadocVersion: 7.12
-    bonitaDockerImageEnterpriseVersion: '7.12.2'
```
Comment (educhastenier) [inline antora.yml:9]: Nice catch, I was about to make a remark on that 😉

File: modules/ROOT/pages/bonita-docker-installation.adoc [hunk 3]
```diff
@@ -7,30 +7,45 @@ This guide assumes you have a working docker environment.
...
 * Information on how to access Quay.io can be found https://customer.bonitasoft.com/download/request[here].
 * You need to provide a valid licence, <<section-StepByStep,follow these steps>> in order to get and configure one.
 ====
 
+
+[#section-versionning]
+
+== Versionning
+
+Multiple tags exists:
```
Comment (educhastenier) [inline modules/ROOT/pages/bonita-docker-installation.adoc]:
```suggestion
Multiple tags exist:
```

**Review by educhastenier — COMMENTED** [review]
Apart from changing the version name from 7.12.2 to 2021.1, what is the purpose of this PR?.

**Comment by baptistemesta** [comment]
> Apart from changing the version name from 7.12.2 to 2021.1, what is the purpose of this PR?.

It explains the difference between the docker tags, see https://bonitasoft.atlassian.net/browse/RUNTIME-289

**Pull request merged**